### ISSUE TITLE
Is it possible to change the orientation of circular tracks?

### ISSUE THREAD

**Issue by flekschas** [gosling-lang/gosling.js#280]
I noticed that circular tracks are by default oriented counterclockwise. Is it possible to change the orientation?

On a side note, I think it might be nice to set the default to clockwise to match circos behavior.

Here's an example: https://gosling-lang.github.io/gosling.js/?full=false&spec=(XassemblyRhg19GXlayoutRcircularGXspacing40%2CXtracksNC*(-'dataK-MurlRhttps%3A%2F%2Fgist.githubusercontent.cU%2Fflekschas%2F1912fdd661851530f7389b18531c0a85%2Fraw%2FHs.hg19.T*ITMHosUeFEH'AMZFYsN'startGLend'%5D-JoverlayNS(SMmarkRrect'A*Mx_startBxe_endBcolorKS*MfEHVS**IOnal'S*)S)-%5DA'styleK'outlineWP40JsizeKD20QKD'gray'QWPKD0.5JyK%20D10%20JwP4500A'height4100C*)C%5D%0A)*%20%20-C**4!%20A%2C-BVIZ'%20)A*MC%0A*D'value4EYRG'%2CHchrUI'typeRJ)A'K4(L%20'M*'N4%5BOnUiPidthQJstrokeR4'S-*Tcsv'AUomVG%20XC'YieldZgeOc_KLfE%01_ZYXVUTSRQPONMLKJIHGEDCBA4-*_

**Comment by sehilyi** [comment]
I agree. We plan to implement the clockwise one, and I think we should use it by default.

(related #76)

**Comment by etowahadams** [comment]
Example of a circular track with counterclockwise axis:

<img width="343" alt="image" src="https://github.com/gosling-lang/gosling.js/assets/14843470/d221928e-c668-42ca-9719-f8ac55bcd518">

```
{
  "layout": "circular",
  "tracks": [
    {
      "width": 400,
      "height": 400,
      "data": {
        "url": "https://resgen.io/api/v1/tileset_info/?d=UvVPeLHuRDiYA3qwFlm7xQ",
        "type": "multivec",
        "row": "sample",
        "column": "position",
        "value": "peak",
        "categories": ["sample 1"],
        "binSize": 5
      },
      "mark": "bar",
      "x": {"field": "start", "type": "genomic", "axis": "bottom"},
      "xe": {"field": "end", "type": "genomic"},
      "y": {"field": "peak", "type": "quantitative", "axis": "right"},
      "size": {"value": 5}
    }
  ]
}
```

**Comment by sehilyi** [comment]
This can be revisited after we work on removing HiGlass dependencies (https://github.com/gosling-lang/gosling.js/issues/1113). Closing this for now.
